h2/utilities.py

```diff
@@ -329,6 +329,13 @@ def _reject_pseudo_header_fields(headers, hdr_validation_flags):
             seen_pseudo_header_fields
         )
 
+    # If ':status' pseudo-header is not there in a response header, reject it
+    if (hdr_validation_flags.is_response_header and
+        not any(field in seen_pseudo_header_fields
+                for field in (b':status', u':status'))):
+        raise ProtocolError(
+            "Couldn't find :status pseudo-header field in response header")
```

Comment:
I'd suggest a slightly different wording, to match that used in `_validate_host_authority_header()`:

> Response header block does not have a :status header.